src/libsystemd/sd-bus/test-bus-watch-bind.c

```diff
@@ -74,13 +74,13 @@ static void* thread_server(void *p) {
         assert_se(fd >= 0);
 
         assert_se(bind(fd, &u.sa, sa_len) >= 0);
-        usleep(100 * USEC_PER_MSEC);
+        (void) usleep(100 * USEC_PER_MSEC);
```

Comment:
Hmm, what's the reason for this? We use `(void)` when *some* of the calls want to ignore the return value, but other don't. When all calls ignore the return value, we don't need `(void)`. Especially in tests, this adds quite a bit of noise.

Let's just drop the few voidifications that are there instead.

Reply:
OK. The change is dropped now.